### DISCUSSION TITLE
Slack notifications

### DISCUSSION THREAD

**Discussion by thejmitchener** [upptime/upptime#78] [Ideas] [answered]
Hello all,

I'm fairly new to Github, Pages, Workflows, etc...

I have a upptime repo setup. Monitoring works flawlessly, I have chosen to use slack for down/up notifications. I have set up the secret successfully, the bot is in the correct channel, I can use the SLACK API to send test messages. However, I do not seem to receive slack notifications as intended via upptime?

Thanks.

**Comment by AnandChowdhary** [comment]
Yes, I'm still working on this. Let's leave this issue open until it's ready. 😄

**Comment by thejmitchener** [comment]
Oh, thank you very much!

**Comment by AnandChowdhary** [comment, marked as answer]
This is ready and tested! 😄

Please see new docs at https://upptime.js.org/docs/notifications!
